app/css/layout.css

```diff
@@ -16,3 +16,8 @@
     max-width: 1440px;
     padding: 0 48px;
 }
+
+.c-react-component {
+    width: fit-content;
+    height: fit-content;
+}
```

Comment:
@SleeplessByte @ErikSchierboom This might go some way to fixing the need for the awkward-as-fuck React vs non-react component problem thing.

Reply:
Ooh, nice!

Reply:
I think [this will be fine](https://caniuse.com/intrinsic-width).